app/javascript/components/mentoring/testimonials-list/RevealedTestimonial.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from 'react'
+import { fromNow } from '../../../utils/time'
+import { Avatar, GraphicalIcon, TrackIcon } from '../../common'
+import { TestimonialModal } from '../../modals/TestimonialModal'
+import { Testimonial } from '../../types'
+
+export const RevealedTestimonial = ({
+  testimonial,
+  isRevealed,
+}: {
+  testimonial: Testimonial
+  isRevealed: boolean
+}): JSX.Element => {
+  const [open, setOpen] = useState(false)
+
+  useEffect(() => {
+    if (!isRevealed) {
+      return
+    }
+
+    setOpen(true)
+  }, [isRevealed])
+
+  return (
+    <React.Fragment>
+      <a
```

Comment:
Feel free to ignore this as I don't know much about accessibility, but is an anchor tag with a `#` href bad for accessibility? Maybe a button is better here? CC @SleeplessByte

Reply:
Updated.